build.go

```diff
@@ -424,5 +474,9 @@ func showProgress(ch chan *controlapi.StatusResponse, noConsole bool) error {
 			c = cf
 		}
 	}
-	return progressui.DisplaySolveStatus(context.TODO(), "", c, os.Stdout, displayCh)
+	out := os.Stdout
+	if stdoutUsed {
+		out = os.Stderr
```

Comment:
can be stderr permanently https://github.com/moby/buildkit/blob/4b7741dc29351b25649fac94fb7920c4bd0e29c9/cmd/buildctl/build.go#L292

Reply:
Nice, changed in d0a4dd70.